components/release.sh

```diff
@@ -43,6 +43,7 @@ images=(
   "resnet-train"
   "ml-pipeline-local-confusion-matrix"
   "ml-pipeline-local-roc"
+  "ml-pipeline-gcp"
```

Comment:
Did you get through the review process for new images?

Reply:
Thanks for the reminder. I just submitted the request.

Reply:
Thanks. Let me know when it is accepted.